### DISCUSSION TITLE
lists of accounts are misplaced and not sufficiently accessible

### DISCUSSION THREAD

**Discussion by aspiers** [rotki/rotki#8933] [Ideas]
## Abstract

Currently the lists of accounts are somewhat hidden as the second section of `Accounts & Balances / Blockchain Balances`, underneath `Blockchain Balances per asset`.  It should be a standalone section accessible via a single click.

## Motivation

There are several problems:

- This section is far too important to be hidden like that.  It's literally the first view I would personally want most of the time (because I'm generally more interested in being able to find assets than constantly check on my wealth like a dragon lol).  So it needs to have its own dedicated view (page).
- For the same reason, it should be possible to reach via one click from the sidebar.
- This section is partially misclassified by its placing under `Blockchain Balances`.  The tables in this section are not just balances.  In fact, the value is only one of many columns (and hopefully even more after #8929).  They are lists of accounts, which happen to include the balances as one feature.
- Ethereum validators are fundamentally different things to normal accounts, so it doesn't make sense to me to group them together with "normal" accounts.

## Specification

I would suggest splitting off the accounts tables into a dedicated page called "Blockchain Accounts", which is linked from the sidebar in the same "Accounts & Balances" section.  Perhaps it could be second, underneath "Blockchain Balances".

**Comment by LefterisJP** [comment]
What exactly is the issue here? I am not sure i follow? The list of accounts is already in its own page. It's blockchain accounts & balances.

This is not an issue and more like a discussion/opinion in which case I would suggest simply using discord to provide said feedback.

**Reply by aspiers** [reply]
> The list of accounts is already in its own page.

It's not?  That's the point:

![image](https://github.com/user-attachments/assets/f286d758-2b27-464e-8ae4-50b89e64fd53)

**Reply by aspiers** [reply]
In other words, it's _part_ of the existing page, underneath a section which in my case is very long, requiring a lot of scrolling to find it.

**Reply by LefterisJP** [reply]
yeah for this we discussed in discord the point is to have accounts at the top of their own view. There we agree.
